crates/forge_loader/src/manifest.rs

```diff
@@ -249,6 +249,42 @@ struct AssetsImportType<'a> {
     import_status: JustFunc<'a>,
 }
 
+// Rovo Modules
+#[derive(Default, Debug, Clone, PartialEq, Eq, Deserialize)] // TODO: this uses all fields for now
+pub struct RovoAgent<'a> {
+    pub key: &'a str,
+    pub name: &'a str,
+    pub description: Option<&'a str>,
+    pub icon: Option<&'a str>,
+    pub prompt: &'a str,
+    #[serde(default, rename = "conversationStarters", borrow)]
+    pub conversation_starters: Vec<&'a str>,
+    #[serde(default, borrow)]
+    pub actions: Vec<&'a str>,
+    #[serde(rename = "followUpPrompt")]
+    pub follow_up_prompt: Option<&'a str>,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize)]
+pub struct ActionInput<'a> {
+    pub title: &'a str,
+    #[serde(rename = "type")]
+    pub input_type: &'a str,
+    pub required: bool,
+    pub description: Option<&'a str>,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize)]
+pub struct Action<'a> {
+    pub key: &'a str,
+    pub name: &'a str, // TODO: this is not in the manifest in the example forge app, wouldn't this fail? https://bitbucket.org/atlassian/rovo-data-discoverability/src/2743f6ca42589c7d1fb2d50e0d9e65e5ef1aebbe/manifest.yml#lines-3
+    pub function: &'a str,
```

Comment:
This should probably be an `Option<&'a str>`, since the action can also just have an `endpoint` [property](https://developer.atlassian.com/platform/forge/manifest-reference/modules/rovo-action/#properties) instead of a `function` property.